calc/compoundInterest.test.ts

```diff
@@ -520,4 +520,30 @@ describe("compoundInterestPerPeriod", () => {
       );
     });
   });
+
+  describe("contributionPerAnnumChange", () => {
+    it("when contributionPerAnnumChange is supplied it increases annual additional contribution amountPerAnnum", () => {
+      const options: IOptions = {
+        type: "contribution",
+        principal: 250_000,
+        rate: 7.8,
+        years: 25,
+        paymentsPerAnnum: 1,
+        amountPerAnnum: 12_000,
```

Comment:
Nice thanks for the update! Last request from me and I will get it merged in 😃 

With the test we want to test the expected output of `amountPerAnnum` is correct, so ideally we would want to test it changes to the correct amount after the first year. You can use `currentPositionInYears` to get the balance for a specific year.

You can do something like this:

```js
const firstYearOptions = { ...initialOptions, currentPositionInYears: 1 };
const secondYearOptions = { ...initialOptions, currentPositionInYears: 2 };

const firstYearResult = compoundInterestPerPeriod(firstYearOptions);
const secondYearResult = compoundInterestPerPeriod(secondYearOptions);
```

Side note: I think I just spotted a previous bug with `accrualOfPaymentsPerAnnum: false` where the end balance is not updating correctly. I am going to raise an issue and a pr for this.

Reply:
actually I just realised we don't actually return `amountPerAnnum` to check the output, should we return this amount so we can see what the final contributions would be further down the line?

Reply:
I am going to approve this and we can add this as an additional feature